requirements/bddtests.txt

```diff
@@ -284,10 +285,11 @@ python-dateutil==2.8.2
     #   celery
     #   faker
     #   mailchimp-transactional
-python-jose[crypto,cryptography]==3.3.0
+python-jose[cryptography]==3.3.0
```

Comment:
We still use python-jose as a indirect dependency, have to also solve that.

Reply:
Yes, working on that, unfortunately I reckon we might need to replace the dependency instead of just removing it.